README.md

```diff
@@ -10,18 +10,18 @@ This module provides an easy way to connect to a Kinvey Business Logic (BL) inst
 
 To install this project, add it to your `package.json` file and install it via `npm`
 ```
-npm install kinvey-backend-sdk
+npm install kinvey-backend-sdk --save
```

Comment:
Remove the `--save` here.  I tend to not suggest it since it sets the `^` flag, which isn't ideal.  The standard in node module documentation is just to specify `npm install <projectname>`